src/Bonsai.ML/Bonsai.ML.csproj

```diff
@@ -4,12 +4,13 @@
     <Title>Bonsai - ML</Title>
     <Description>Bonsai Library containing reactive infrastructure for machine learning.</Description>
     <PackageTags>Bonsai Rx ML Machine Learning</PackageTags>
-    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
     <VersionPrefix>0.1.0</VersionPrefix>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Bonsai.Core" Version="2.8.0" />
+    <PackageReference Include="Bonsai.Core" Version="2.8.1" />
+    <PackageReference Include="pythonnet" Version="3.0.3" />
```

Comment:
It makes sense to have reusable Python.NET functionality across python package, but I'm wondering whether we should have this on the top core package, or in a downstream `Bonsai.ML.Python` package.

The reason is that we might have in the future either Julia or R packages and if you are using one of those you might not be interested in taking Python.NET as dependency.

Reply:
Were going to move the python helper classes into the lds package for now. This will avoid us from committing to a specific implementation. Revisit in the future